docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-runtime.adoc

```diff
@@ -56,6 +56,24 @@ The following enhancements are planned; the details will be available at release
 |*Exclude Windows vulnerabilities found in Go packages from UNIX OS*
 |Prisma Cloud now excludes vulnerabilities found in Go packages that are specific to Windows from UNIX based operating systems in the Vulnerability Explorer. Select *Monitor > Vulnerabilities > Vulnerability Explorer* for a comprehensive list of the detected vulnerabilities.
 
+//CWP-57947
+//TODO:Pending blurb approval for Kamesh
+|*Support for Google registry scanning*
+|When onboarding GCP cloud accounts, Prisma Cloud now supports scanning of Google Container Registry (GCR) and Google Artifact Registry (GAR).
```

Comment:
Suggest a minor rewrite to avoid starting the sentence with a conditional phrase.

Prisma Cloud supports the scanning of Google Container Registry (GCR) and Google Artifact Registry (GAR), when onboarding GCP cloud accounts.